bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/service.js

```diff
@@ -0,0 +1,82 @@
+import Presentations from '/imports/api/2.0/presentations';
+import Auth from '/imports/ui/services/auth';
+import { call } from '/imports/ui/services/api';
+
+const futch = (url, opts = {}, onProgress) => new Promise((res, rej) => {
```

Comment:
perhaps `fetch`?

Reply:
I was quite confused by this so I looked it up. At first I thought it was a typo, and then I thought maybe it stood for File Upload fetch. From what Ive seen online it's a very rare name for XHR file upload requests. Because in certain cases you would need it part of the XHR/ajax object but if you named it fetch it would overwrite the original method name so people went with `futch`. AFAIK since this is a function (not part of a class) it could just be named fetch since there's no conflict. Perhaps Oswaldo went with this just to keep it separated from fetch. I think a comment explaining it would definitely be helpful. 

[See here](https://github.com/g6ling/React-Native-Tips/tree/master/How_to_upload_photo%2Cfile_in%20react-native#how-can-i-add-progress)

Reply:
It is exactly what @perroned said. Using `fetch` API its not possible to do a upload progress so instead we use a wrapper `futch` that Promisify the `XMLHttpRequest` and add a way to fire `onprogress` events.

Reply:
I did a quick search before commenting last night but did not read as far as @perroned (kudos!)!
An inline comment with the explanation you gave here would be _very_ much appreciated!